src/ProgressiveCurve.sol

```diff
@@ -63,14 +63,14 @@ contract ProgressiveCurve is BaseCurve {
         require(slope18 > 0, "PC: Slope must be > 0");
 
         SLOPE = UD60x18.wrap(slope18);
-        HALF_SLOPE = SLOPE.div(UD60x18.wrap(2));
```

Comment:
3.1 - Wrong Use of PRB-Math Library in Progressive Curves
Same change here as with OffsetProgressiveCurve.  We adjusted this to properly get half slope by dividing by 2.